### PR TITLE
Model updates from 20210728

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/models/risk.py [hunk 1]
```diff
@@ -144,6 +147,25 @@ def getModelDefs(self):
                     }),
                 )),
 
+                ('risk:alert', {}, (
```
Comment (reign-git) [inline synapse/models/risk.py:150]: consider adding 'name' prop to capture the name of the alert.

Comment (reign-git) [inline synapse/models/risk.py:150]: How will the alert be linked back to an IP (for network alerts) or a host (for host-based alerts)?

Comment (reign-git) [inline synapse/models/risk.py:150]: Do we want to track what generated the alert? For example this software/appliance/host sent this alert?

Comment (invisig0th) [reply synapse/models/risk.py:150]: light edges for now

Comment (invisig0th) [reply synapse/models/risk.py:150]: For flexibility, i'd prefer to use light edges for this for a while as well until we know what fits best :)

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**